downstream/modules/platform/proc-hub-ingress-options.adoc

```diff
@@ -4,7 +4,40 @@
 
 The {PlatformName} operator installation form allows you to further configure your {ControllerName} operator Ingress under *Advanced configuration*.
 
+.Procedure
+
 . Click *Advanced Configuration*.
 . Under *Ingress type*, click the drop-down menu and select *Ingress*.
 . Under *Ingress annotations*, enter any annotations to add to the ingress.
-. Under *Ingress TLS secret*, click the drop-down menu and select a secret from the list.
\ No newline at end of file
+. Under *Ingress TLS secret*, click the drop-down menu and select a secret from the list.
+
+After you have configured your {HubName} operator, click *Create* at the bottom of the form view. {OCP} will now create the pods. This may take a few minutes.
+
+You can view the progress by navigating to menu:Workloads[Pods] and locating the newly created instance.
+
+.Verification
+
+Verify that the following operator pods provided by the {PlatformNameShort} Operator installation from {HubName} are running:
+[cols="a,a,a"]
+|===
+| automation-controller-operator-controller-manager | {HubName} | {ControllerName}
+
+|
+* automation-hub-operator-controller-manager
+* Resource-operator-controller-manager
+|
+* controller
+* controller-postgres
+|
+* Api
+* <deployment-name>-ui
+* Content
+* Worker
+* Postgres
```

Comment:
There are a few changes needed here.  

```
$ oc get pods
NAME                                                              READY   STATUS    RESTARTS        AGE
automation-controller-operator-controller-manager-7768f946f4f4b   2/2     Running   0               15m
automation-hub-operator-controller-manager-79447766cf-q7wj5       2/2     Running   0               15m
resource-operator-controller-manager-78677c579-64ldr              2/2     Running   0               15m
```

These are all of the operator manager controllers for each of the 3 operators (controller, hub, resource operators)

After deploying Controller, you will see the addition of these pods:

```
$ oc get pods
NAME                                                              READY   STATUS    RESTARTS        AGE
controller-7f5bff64bf-xc9cm                                       4/4     Running   3 (8m33s ago)   13m
controller-postgres-13-0                                          1/1     Running   0               14m
```

And after deploying Hub, you will see the alddition of these pods:

```
$ oc get pods
NAME                                                              READY   STATUS    RESTARTS        AGE
hub-api-6b74cbfd79-gp7l9                                          1/1     Running   1 (10m ago)     13m
hub-content-76c578c9fc-dfnhv                                      1/1     Running   0               13m
hub-content-76c578c9fc-xdtkf                                      1/1     Running   0               13m
hub-postgres-13-0                                                 1/1     Running   0               14m
hub-redis-7d65d8479d-8n6mk                                        1/1     Running   0               14m
hub-worker-dbb6fd7bd-bsp8b                                        1/1     Running   0               13m
hub-worker-dbb6fd7bd-qtk8k                                        1/1     Running   0               13m
```

Reply:
@rooftopcellist, I have implemented the changes based on your feeback above. For hub, I only included 1 mention of the content and worker nodes to avoid duplication. If you think that will cause confusion, let me know and I can include them. If you're happy with the changes, please approve so I can get this published and closed out. Thanks in advance! ~ Donna